src/features/environment.js

```diff
@@ -143,7 +146,17 @@ export const EnvironmentMixin = (ModelViewerElement) => {
       this[$setShadowLightColor](parsedColor);
 
       // TODO(#336): can cache this per renderer and color
-      const environmentMap = textureUtils.generateDefaultEnvironmentMap();
+      const environmentMap = textureUtils.generateDefaultEnvironmentMap({
+        pmrem: this.experimentalPmrem
+      });
+
+      this[$applyEnvironmentMap](environmentMap);
+    }
+
+    /**
+     * @param {THREE.Texture} environmentMap
```

Comment:
Above, you invoke this method with `null` as the argument. Can we document what this is intended to do?

Reply:
:+1: